examples/meteringlab/c2/cassandra-stress.yaml

```diff
@@ -0,0 +1,40 @@
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cassandra-stress-0
```

Comment:
it's just one job taking resources of entire node, you need to send send traffic to several scylla clusters.